### PR TITLE
feedback from pr61

### PR DESCRIPTION
**Pull request by FoamyGuy**
Handles all items noted in the comments in #61.

@tekktrik you mentioned in the other thread wanting to take another look after those things were changed. Once you have a chance to do that let me know and I can add any further changes to this branch.

The one change in this that wasn't mentioned was the removal of: 
```
# pylint: disable=consider-using-f-string
```
I found that pylint is passing even without that so I removed it.

### CODE REVIEW

File: adafruit_bme680.py [hunk 1]
```diff
@@ -537,86 +502,75 @@ def _set_heatr_conf(self, heater_temp: UINT16, heater_time: UINT16) -> None:
...
-            self._write(ctrl_gas_addr_0, [ctrl_gas_data_0])
-            self._write(ctrl_gas_addr_1, [ctrl_gas_data_1])
+            self._write(_BME68X_REG_CTRL_GAS_0, [ctrl_gas_data_0])
+            self._write(_BME68X_REG_CTRL_GAS_1, [ctrl_gas_data_1])
             # HELP check this
             self._set_op_mode(_BME68X_FORCED_MODE)
-        except GasHeaterException as exc:
+        except OSError as exc:
             self._set_op_mode(_BME68X_FORCED_MODE)
             raise exc
```
Comment (tekktrik) [inline adafruit_bme680.py]: I think this might be simplified to:

```suggestion
        finally:
            self._set_op(_BME68X_FORCED_MODE)
```

**Review by tekktrik — CHANGES_REQUESTED** [review]
One small suggestion I don't think I picked up on originally, but I think this looks good!  Thanks for implementing all of these!

File: adafruit_bme680.py (hunk 1 above)
Comment (FoamyGuy) [reply adafruit_bme680.py]: I tried this but pylint reports this error:
```
adafruit_bme680.py:509:12: E1101: Instance of 'Adafruit_BME680' has no '_set_op' member (no-member)
```

I'm guess that it's meant to be `_set_op_mode()` I'll commit it with that change.

**Comment by FoamyGuy** [comment]
The latest commit adopts the requested change. Take another look when you have a chance @tekktrik

**Comment by FoamyGuy** [comment]
I'm going to merge this based on the original 'looks good' other than the addressed change. We can always add or change things in the future if anything else comes up.

**Pull request merged**